src/javascript/app_2/App/Components/Elements/Calendar/calendar_header.jsx

```diff
@@ -24,7 +24,7 @@ const CalendarHeader = ({
     isPeriodDisabled,
     navigateTo,
     switchView,
-    disabled_selector,
+    disabled_selector = [],
```

Comment:
please consider moving disabled_selector before isPeriodDisabled, in the matter of sorting props alphabetically